src/server/rest/v1/service/PricingService.ts

```diff
@@ -0,0 +1,175 @@
+import { Action, Entity } from '../../../../types/Authorization';
+import { NextFunction, Request, Response } from 'express';
+
+import AppAuthError from '../../../../exception/AppAuthError';
+import AuthorizationService from './AuthorizationService';
+import Constants from '../../../../utils/Constants';
+import { HTTPAuthError } from '../../../../types/HTTPError';
+import Logging from '../../../../utils/Logging';
+import { PricingDataResult } from '../../../../types/DataResult';
+import PricingDefinition from '../../../../types/Pricing';
+import PricingStorage from '../../../../storage/mongodb/PricingStorage';
+import PricingValidator from '../validator/PricingValidator';
+import { ServerAction } from '../../../../types/Server';
+import { TenantComponents } from '../../../../types/Tenant';
+import UtilsService from './UtilsService';
+
+const MODULE_NAME = 'PricingService';
+
+export default class PricingService {
+
+  public static async handleGetPricingDefinition(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
+    // Check if component is active
+    UtilsService.assertComponentIsActiveFromToken(req.user, TenantComponents.PRICING,
+      Action.READ, Entity.PRICING_DEFINITION, MODULE_NAME, 'handleGetPricingDefinition');
+    // Filter
+    const filteredRequest = PricingValidator.getInstance().validatePricingDefinitionGet(req.query);
+    UtilsService.assertIdIsProvided(action, filteredRequest.ID, MODULE_NAME, 'handleGetPricingDefinition', req.user);
```

Comment:
To remove, this is checked already in UtilsService.checkAndGetPricingDefinitionAuthorization()